### ISSUE TITLE
Support for Organic Installs on Stitch

### ISSUE THREAD

**Issue by BernalCarlos** [singer-io/tap-appsflyer#35]
Hello Everyone,

I see that the AppsFlyer tap is now capable of syncing organic installs and that it's now officially tagged on the master branch.
https://github.com/singer-io/tap-appsflyer/pull/34

However this new capability is behind a CONFIG variable, that is not available on the Stitch integration.

Does anyone know how can I make this config value available on the Stitch integration, or if there is a workaround in order to get this working on Stitch?

Thank you

**Comment by iambizan** [comment]
Hi @BernalCarlos, were you able to import the organic data from appsflyer?

**Comment by BernalCarlos** [comment]
No, I didn't. Basically, we had to get that data from scratch using a custom script.
